optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingDomainModel/CloudBalancingDomainModel-section.adoc

```diff
@@ -8,25 +8,30 @@
 
 Using a <<domainModelingGuide,domain model>> helps determine which classes are planning entities and which of their properties are planning variables. It also helps to simplify contraints, improve performance, and increase flexibility for future needs. 
 
+To create a domain model, define all the objects that represent the input data for the problem. In this simple example, the objects are processes and computers. 
+
+A separate object in the domain model must represent a full data set of problem, which contains the input data as well as a solution. In this example, this object holds a list of computers and a list of processes. Each process is assigned to a compter; the distribution of processes between computers is the solution.
+
 . Draw a class diagram of your domain model. 
 . Normalize it to remove duplicate data.
-. Write down some sample instances for each class. 
+. Write down some _sample instances_ for each class. Sample instances are entity properties that are relevant for planning purposes.
 
 * ``Computer``: represents a computer with certain hardware and maintenance costs. 
 +
 In this example, the sample instances for the `Computer` class are: `cpuPower`, `memory`, `networkBandwidth`, `cost`. 
 * ``Process``: represents a process with a demand. Needs to be assigned to a `Computer` by Planner. 
 +
 Sample instances for ``Process`` are: `requiredCpuPower`, `requiredMemory`, and `requiredNetworkBandwidth`.
-* ``CloudBalance``: represents a problem. Contains every `Computer` and `Process` for a certain data set. 
+* ``CloudBalance``: represents the distribution of processes between computers. Contains every `Computer` and `Process` for a certain data set. 
 +
-The sample instance for `CloudBalance` is `score`.
+For an object representing the full data set and solution, a sample instance holding the _score_ must be present. Planner can calculate and compare the scores for different solutions; the solution with the highest score is the optimal solution. Therefore, the sample instance for `CloudBalance` is `score`.
 
 . Determine which relationships (or fields) change during planning.
 
-* Planning entity: the class (or classes) that changes during solving. In this example, it is the class ``Process``.
-* Planning variable: the property (or properties) of a planning entity class that changes during solving. In this example, it is the property `computer` on the class ``Process``.
-* Planning solution: the class that represents a data set and contains all planning entities. In this example that is the class ``CloudBalance``.
+* _Planning entity_: the class (or classes) that Planner can change during solving. In this example, it is the class ``Process``, because Planner can move processes to different computers.
```

Comment:
can move processes to different computer -> can assign processes to computers

they're not always different. They start out unassigned, so "assign" is more accurate than "move".

Reply:
changed, thanks